documentation/devonfw-ide-introduction.asciidoc

```diff
@@ -1,12 +1,73 @@
-= Introduction
+= devonfw-ide
 
-http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries
-as well as industry proven practices and code conventions.
+== What is devonfw-ide?
+
+`devondw-ide` is a standardized, open-source toolbox or construction kit allowing you to pick and choose those tools which fit the requirements and needs of your project.
```

Comment:
This is too long and has too big buzzwords to really engage the user

Reply:
simplified.